frontend/src/Menu/index.jsx

```diff
@@ -1,6 +1,8 @@
-import React, { useState } from 'react';
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
 import { View, Text, Dimensions } from 'react-native';
 import { TouchableImage } from '../common/components';
+import { switchMenu } from '../redux/actionCreator';
```

Comment:
action name is not descriptive. You are not switching the menu. It should be changeRenderedComponent, or changeDisplayedComponent or something similar